org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/annotations/TypeAnnotation.xtend

```diff
@@ -17,12 +18,18 @@ class SimpleTypeAnnotation<T extends WollokType> implements TypeAnnotation {
 }
 
 class ClassParameterTypeAnnotation implements TypeAnnotation {
+	@Accessors(PUBLIC_GETTER)
```

Comment:
A class parameter annotation became a bit more subtle, as it now is more than "a string" (i.e. the param name). It is also associated to the container type.